src/Compilers/Core/CodeAnalysisTest/CryptoBlobParserTests.cs

```diff
@@ -74,11 +166,10 @@ public void TryGetPublicKeyFailsForInvalidKeyBlobs()
                 "0602000000240000DEADBEEF" + new string('0', 136 * 2), // public key blob without magic public key
             };
 
-            ImmutableArray<byte> pubKey;
-            foreach (var key in invalidKeyBlobs)
-            {
-                Assert.False(CryptoBlobParser.TryGetPublicKey(HexToBin(key), out pubKey));
-            }
+            Assert.False(CryptoBlobParser.TryParseKey(HexToBin(invalidKeyBlobs[0]), out _, out _));
```

Comment:
when the test fails, I can just double-click on the exception message and it'll tell me exactly which test-case was the one that broke.